.github/workflows/rust.yml

```diff
@@ -27,8 +27,19 @@ jobs:
       run: |
         apt-get update && apt-get install -y curl
         apt-get install build-essential -y
-    - uses: actions-rs/toolchain@v1
+    - name: Run test with Rust nightly
+      uses: actions-rs/toolchain@v1
       with:
+        profile: minimal
+        toolchain: nightly
+        override: true
+      on:
+          schedule:
+            - cron: '0 0 * * *' # Run daily at midnight (UTC)
```

Comment:
I did not know we could do this 😱
I don't find this in the GitHub docs, can you show me the link or proof this is working in one of your repo? 😇

Reply:
I think your solution @hugoamalric does not work

https://github.com/meilisearch/meilisearch/actions/runs/4133694844

<img width="959" alt="Capture d’écran 2023-02-09 à 12 28 38" src="https://user-images.githubusercontent.com/20380692/217800692-97f609c0-196f-42c1-878d-16a1f091b635.png">

Reply:
Yes my bad I thought it was possible to do so. 

The cron job is in fact scoped to the whole workflow. Is it annoying if the whole workflow is triggered every day at midnight? Otherwise, it is also possible to declare a condition for each job so that they are not affected by this cron job.